src/it/scala/RollingUpgradeSpec.scala

```diff
@@ -55,7 +55,12 @@ class RollingUpgradeSpec
   override implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(180, Seconds), interval = Span(5, Seconds))
 
   // add nightly here before a release
-  val akkaVersions = Seq("2.5.23", "2.5.26", "2.6.0")
+  val akkaVersions = {
+    Seq("2.5.23", "2.5.29") ++
+    Option(System.getProperty("cron.build.akka.25.snapshot")).toSeq ++
+    Seq("2.6.3")++
+    Option(System.getProperty("cron.build.akka.26.snapshot")).toSeq
+  }
```

Comment:
Bumped `2.5.29` and `2.6.3` and interleaved the snapshots resolved from bash.

We should consider automating the resolution of `2.5.<latest>` and `2.6.<latest>` too.